app/views/gardens/index.html.haml

```diff
@@ -22,43 +21,38 @@
 
   %table.table.table-striped
     %tr
+      %th 
       - unless @owner
         %th Owner
       %th Garden name
-      %th Description
-      %th Location
       %th Area
-      %th Active?
+      %th Location
       %th Plantings
-      %th
 
     - @gardens.each do |garden|
       %tr
+        
+        %td.status
```

Comment:
yes, it would be nice to put a unique id in the css in each page, like #garden_index.
In that way after in less we could use :
#garden_index {
   .garden-status {
   }
}

definetly even the organisation of the less file is far away to be perfect.